### ISSUE TITLE
a11y: Dark mode issues

### ISSUE THREAD

**Issue by illume** [kubernetes-sigs/headlamp#3746]
I spotted a few contrast issues in dark mode...

1) This one on the Notifications action menu:

<img width="380" height="316" alt="Image" src="https://github.com/user-attachments/assets/2ff62204-eac0-4f96-81ee-fcb70bbf5130" />

2) I noticed also the "Cancel" button on actions is unreadable

<img width="747" height="244" alt="Image" src="https://github.com/user-attachments/assets/31749d81-bc45-4aa5-bf57-09ccdad4cac4" />

3) The issue pasted in slack headlamp channel today was similar to one I saw in my plugin where `<Alert severity="warning">` has text that's too dark.

<img width="1229" height="125" alt="Image" src="https://github.com/user-attachments/assets/709261ea-e848-48e7-bb48-e40c443eb62e" />

**Comment by vyncent-t** [comment]
working on a fix

**Comment by illume** [comment]
@vyncent-t thanks kindly

**Comment by illume** [comment]
I think these are all fixed now 🎉

@vyncent-t please reopen if I'm wrong.
